scrapy_poet/utils.py

```diff
@@ -100,14 +100,14 @@ def is_provider_using_response(provider):
     for argument, possible_types in andi.inspect(provider.__init__).items():
         for cls in possible_types:
             if not issubclass(cls, Response):
-                # Type annotation is not a sub-class of Response.
+                # Type annotation is not a subclass of Response.
                 continue
 
             if issubclass(cls, DummyResponse):
```

Comment:
Do we still need to support DummyResponse here?

Reply:
It seems that if we drop DummyResponse support, is_provider_using_response function can be simplified